components/fxa-client/src/http_client.rs

```diff
@@ -491,6 +518,12 @@ pub struct OAuthTokenResponse {
     pub access_token: String,
 }
 
+#[derive(Deserialize)]
+pub struct IntrospectResponse {
+    pub active: bool,
+    pub token_type: String,
```

Comment:
We don't have to include *all* the fields from the `/introspect` response, but please include `scope`, `exp` and `iss` in this list for the first version. (Both here, and in `IntrospectInfo` message above).